java/configuration/src/main/java/sleeper/configuration/properties/instance/CommonProperty.java

```diff
@@ -155,6 +155,14 @@ public interface CommonProperty {
             .validationPredicate(Utils::isNonNullNonEmptyString)
             .propertyGroup(InstancePropertyGroup.COMMON)
             .runCdkDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty METRICS_LAMBDA_CONCURRENCY_RESERVED = Index.propertyBuilder("sleeper.metrics.concurrency.reserved")
+            .description("The reserved concurrency for the table metrics lambda.")
+            .defaultValue(null)
+            .propertyGroup(InstancePropertyGroup.COMMON).build();
+    UserDefinedInstanceProperty METRICS_LAMBDA_CONCURRENCY_MAXIMUM = Index.propertyBuilder("sleeper.metrics.concurrency.max")
+            .description("The maximum given concurrency allowed for the table metrics lambda.")
```

Comment:
It would be good to show what the distinction is between reserved and maximum concurrency, and to clarify that the maximum concurrency is set on the event source rather than the lambda. A link to the AWS documentation would be really great.

Reply:
Addressed